scripts/install/opt/seagate/cortx/motr/bin/motr_mini_prov.py

```diff
@@ -179,6 +208,9 @@ def configure_lnet(self):
         fp.write(lnet_config)
 
     restart_services(self, ["lnet"])
+    # Ping to nid
+    sys.stdout.write("Doing ping to nids\n")
+    lnet_self_ping(self)
```

Comment:
Do we need to return the rc of lnet_self_ping() to caller?

Reply:
Done 19c9afc